### ISSUE TITLE
Update README.md

### ISSUE THREAD

**Issue by VikasPandey121** [VikasPandey121/WildCare#14]
Your task is to :

- Go through the README.md File and lookup for the grammatical error or any missing information 
- Attach screenshots of all screens in a proper format in the README.md file

**Comment by LadySqrrl** [comment]
I can edit the README.md file and add screenshots of all the steps if you still need that. I forked and cloned it before reading thoroughly and didn't comment first. It's one of my first contributions so I'm still learning how to do them. I'm sorry!

**Comment by VikasPandey121** [comment]
> I can edit the README.md file and add screenshots of all the steps if you still need that. I forked and cloned it before reading thoroughly and didn't comment first. It's one of my first contributions so I'm still learning how to do them. I'm sorry!

GO AHEAD AND MAKE THIS README FILE MORE INTERACTIVE. DON'T FORGET TO MENTION THE ISSUE NUMBER IN YOUR PR. 👍🏻

**Comment by aakash232** [comment]
@VikasPandey121 
The Readme file doesn't have screenshots for the app. 
A more detailed content for the Features and Tech used in a creative way can go great for both developers and users in the repo. 
If still open, I'd like to add some beautified and tech-oriented addOns

**Comment by VikasPandey121** [comment]
Yes @aakash232, This one is an open issue. You can begin your contribution to it.
